src/ruleset/v2/functions/messageExamples.ts

```diff
@@ -6,6 +6,23 @@ import type { IFunctionResult, RulesetFunctionContext } from '@stoplight/spectra
 import type { JSONSchema7 } from 'json-schema';
 import type { v2 } from 'spec-types';
 
+function serializeSchema(schema: unknown, type: 'payload' | 'headers'): any {
+  if (!schema && typeof schema !== 'boolean') { // if schema is falsy then
+    if (type === 'headers') { // object for headers
+      schema = { type: 'object' };
+    } else { // anything for payload
+      schema = {};
+    }
+  } else if (typeof schema === 'boolean') { // spectral cannot handle boolean schemas
+    if (schema === true) {
+      schema = {}; // everything
+    } else {
+      schema = { not: {} }; // nothing
+    }
+  }
+  return schema;
```

Comment:
Wtf 😆

Reply:
You only can pass object as schema, yeah weird but it's easier to handle that in this way rather than contribution :P